homebrew/Brewfile

```diff
@@ -1,88 +1,81 @@
 # frozen_string_literal: true
 
-tap 'homebrew/bundle'
-tap 'homebrew/core'
 tap 'homebrew/cask'
-tap 'homebrew/services'
 tap 'homebrew/cask-fonts'
+tap 'homebrew/services'
 tap 'pmeinhardt/tools'
 
-# apps
+# core apps
+cask 'firefox'
 cask 'flux'
-cask 'rectangle'          # window management
+cask 'iterm2'
 cask 'karabiner-elements' # keyboard mappings
+cask 'keepassxc'          # password manager
 cask 'nextcloud'
-cask 'whatsapp'
-cask 'signal'
-cask 'telegram'
+cask 'rectangle'          # window management
 
-cask 'chromedriver'
-cask 'google-chrome'
-cask 'iterm2'
+# dev apps
 cask 'atom'
-cask 'java'
-cask 'insomnia'           # postman alternative
+cask 'chromedriver'
 cask 'docker'
+cask 'google-chrome'
 
-cask 'keepassxc'          # password manager
-cask 'firefox'
+# other apps
+cask 'gimp'
+cask 'imageoptim'         # remove exif-data from images
+cask 'inkscape'
 cask 'kap'                # record screencasts
 cask 'libreoffice'
+cask 'signal'
 cask 'slack'
-cask 'gimp'
-cask 'inkscape'
-cask 'imageoptim'         # remove exif-data from images
-
 cask 'spotify'
+cask 'telegram'
 cask 'vlc'
+cask 'whatsapp'
 
 # fonts
 cask 'font-dosis'
-cask 'font-lato'
 cask 'font-quicksand'
-cask 'font-source-sans-pro'
 
-# Unix tools
+# packages
+brew 'autoconf'            # for installing erlang
 brew 'coreutils'           # GNU file, shell and text utilities
 brew 'gpg'                 # GNU privacy packages
+brew 'htop'
 brew 'openssl'
 brew 'readline'
+brew 'rsync'
+brew 'tree'
 brew 'watch'               # execute command periodically
 brew 'wget'
-brew 'rsync'
-brew 'autoconf'            # for installing erlang
 brew 'wxmac'               # for starting erlang observer / debugger
 
-# dev tools
+# dev packages
+brew 'ansible'
+brew 'fd'
 brew 'fzf'
-brew 'tmux'
-brew 'reattach-to-user-namespace' # fix tmux for macOS
-brew 'the_silver_searcher'        # the Ag search tool
-brew 'overmind'                   # Process manager (Procfile based)
-brew 'shellcheck'                 # Linter for shell scripts
-brew 'python'
-brew 'git'
 brew 'gh'                         # Github CLI
-brew 'neovim'
-brew 'zsh'
-brew 'htop'
-brew 'fd'
-brew 'ansible'
+brew 'git'
+brew 'hadolint'                   # lint dockerfiles
 brew 'jq'                         # format JSON
+brew 'neovim'
+brew 'overmind'                   # Process manager (Procfile based)
+brew 'python'
+brew 'reattach-to-user-namespace' # fix tmux for macOS
+brew 'shellcheck'                 # Linter for shell scripts
+brew 'the_silver_searcher'        # the Ag search tool
```

Comment:
Randomly browsing through the diff… I've switched from `ag` to [`rg`](https://github.com/BurntSushi/ripgrep) some time ago and would recommend to check it out. ✌️